operations/iuf/workflows/management_rollout.md

```diff
@@ -367,6 +306,52 @@ Once this step has completed:
 Return to the procedure that was being followed for `management-nodes-rollout` to complete the next step, either [Management-nodes-rollout with CSM upgrade](#31-management-nodes-rollout-with-csm-upgrade) or
 [Management-nodes-rollout without CSM upgrade](#32-management-nodes-rollout-without-csm-upgrade).
 
+## 3.4 Personalize NCN Storage Nodes
+
+> **`NOTE`**
+> A customized image is created for NCN storage nodes during the prepare images stage. That image is the same image that is running on NCN storage nodes so there is no need to 'upgrade' into that image.
+> However, if it is desired to rollout the NCN storage nodes into the customized, this can be done by following [upgrade NCN storage nodes into the customized image](../stages/management_nodes_rollout.md#upgrade-ncn-storage-nodes-into-the-customized-image).
```

Comment:
```suggestion
> However, if it is desired to rollout the NCN storage nodes with the customized image, this can be done by following [upgrade NCN storage nodes into the customized image](../stages/management_nodes_rollout.md#upgrade-ncn-storage-nodes-into-the-customized-image).
```